src/set_api.js

```diff
@@ -1,7 +1,6 @@
 const BASE_URL = 'https://api.spaceflightnewsapi.net/v3/articles';
 const involvementApi = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
 const commentsUrl = '/apps/Tl04OrbFyu6fxKeYxZ5Q/comments';
-// const likesUrl = '/apps/Uinv3HFYXneji5y5pk1g/likes';
```

Comment:
-[ ] Optional
- Please remove unused code here and to all other places where you have it in your codes.  Unused commented code makes your code not clean